languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,54 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_QUE" name="quelle que" default="temp_off">
+          <rule>
+              <pattern>
+                  <marker>
+                      <token>qu'</token>
```

Comment:
You could use the apostrophe entity here.

By the way, in some rules, you match `qu'?`, in some you match `qu'` – which one do you think is better? Please apply your decision to all QUELLES rules.